Extend/Controller/Admin/PaymentMain.php

```diff
@@ -215,7 +225,65 @@ private function _validateRequestParameters($aParams)
             return $oTransactionService->checkCredentials();
         }
 
-        return false;
+        // handle the case of different configuration per currency (currently only Payolution)
+        // there it's only checked if config values are entered on the frontend but no validation is done on the backend
+        return $this->_checkCurrencyConfigFields($aParams);
+    }
+
+    /**
+     * Checks if the config fields for all selected allowed currencies have been set.
+     *
+     * @param array $aParams
+     *
+     * @return bool
+     *
+     * @since 1.2.0
+     */
+    private function _checkCurrencyConfigFields($aParams)
+    {
+        $aOldCurrencyValue = $this->_getPaymentMethod()->oxpayments__allowed_currencies->value ?? [];
+        $aNewCurrencyValue = $aParams['oxpayments__allowed_currencies'];
```

Comment:
At the moment it is not possible that this happens because the frontend input field has the `required` attribute, so the merchant must select at least one currency.